starter_code/components.js

```diff
@@ -0,0 +1,41 @@
+class Picture extends React.component {
+  constructor(){
+    super();
+    this.state = { image: 'http://images.fanpop.com/images/image_uploads/wooderson---Slater-dazed-and-confused-301811_400_300.jpg'}
+  }
+  _revealMyImage(image) {
+    let currentImage = this.state.image;
+
+    if (currentImage == 'http://images.fanpop.com/images/image_uploads/wooderson---Slater-dazed-and-confused-301811_400_300.jpg') {
+      this.setState({ image: ''})
+    } else this.setState({ image: 'http://images.fanpop.com/images/image_uploads/wooderson---Slater-dazed-and-confused-301811_400_300.jpg'})
+  }
+  render() {
+    return (
+      <div>
+        <p> There is a picture somewhere...<p>
+        <img src={ this.state.image }/>
```

Comment:
Now that our state is saved as a boolean state, we can show or hide this image entirely

```js
{ this.state.image ? <img src='http://images.fanpop.com/images/image_uploads/wooderson---Slater-dazed-and-confused-301811_400_300.jpg'/> : null}
```